frontend/src/service/ChangeLog.service.ts

```diff
@@ -110,7 +126,7 @@ export class ChangeLogService {
     return new Promise(async (resolve, reject) => {
       this.changeLogGraphQLApi
         .getAllChanges()
-        .then(resolve)
+        .then(changes => resolve(this.sortChanges(changes)))
```

Comment:
I wonder why this is needed?

Reply:
@byliuyang The `getAllChanges` is not returning changes in the same order when some updates happen. It's better if we follow some intiutive order while showing for the admins to moderate.